modules/renter/contractor/contractor.go

```diff
@@ -85,14 +85,12 @@ func (c *Contractor) Contract(hostAddr modules.NetAddress) (modules.RenterContra
 	return modules.RenterContract{}, false
 }
 
-// Contracts returns the contracts formed by the contractor.
+// Contracts returns the contracts formed by the contractor. Only contracts
+// formed with currently online hosts are returned.
 func (c *Contractor) Contracts() (cs []modules.RenterContract) {
 	c.mu.RLock()
 	defer c.mu.RUnlock()
-	for _, c := range c.contracts {
-		cs = append(cs, c)
-	}
-	return
+	return c.onlineContracts()
```

Comment:
it's technically correct here but strict conventions make it easier to modify in the future